libvast/test/system/importer.cpp

```diff
@@ -94,9 +94,10 @@ struct importer_fixture : Base {
   virtual void fetch_ok() = 0;
 
   auto make_source() {
-    return vast::detail::spawn_container_source(this->self->system(),
-                                                this->bro_conn_log_slices,
-                                                importer);
+    auto slices = this->copy(this->bro_conn_log_slices);
+    vast::detail::spawn_container_source(this->self->system(), slices,
```

Comment:
This function previously returned the *actor*, not the slices. We can also move `slices` into the container source to avoid a superfluous copy.